### ISSUE TITLE
Style issue for clear button

### ISSUE THREAD

**Issue by SkripalBoris** [ericgio/react-bootstrap-typeahead#287]
### Version
2.0.2
### Steps to reproduce
Example:
```ts
 <Typeahead
    clearButton={true}
    labelKey="name"
    selected={[{name: "This is very-very-very long string that break clear button display when container is not so long and have fixed width for example"}]}
    options={[{name: "This is very-very-very long string that break clear button display when container is not so long and have fixed width for example"}]}
  />
```
### Expected Behavior
Good autocomplete with a nice clear button on the right side of field
### Actual Behavior
Good autocomplete but without a nice clear button on right side of the field, because text input has fixed width and this width is more than parent container and nice clear button is pushed far to the right and hidden by `overflow: hidden`

**Comment by ericgio** [comment]
Thanks for the report. Should be fixed as of v2.0.3

**Comment by SkripalBoris** [comment]
Thanks for the fix
